ChangeLog.md

```diff
@@ -1,8 +1,9 @@
 # ICPC Tools Changelog
 
-## V2.2.0 - February, 2021
+## V2.2.0 - February 6, 2021
 -----------------
-* Move to Github and Gitlab.
+* First open source release!
```

Comment:
Should we call it “open” instead of “open source”? Since technically it’s not open source in the official sense